pkg/acsengine/testdata/location/kubernetes.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
I use locality in the certname. For cases not provide location, it will just use "", the empty string.